src/probeinterface/io.py

```diff
@@ -1700,6 +1700,15 @@ def read_openephys(
             raise Exception("NP_PROBE field not found in settings")
         return None
 
+    # In neuropixel plugin 0.7, the option for enabling/disabling probes was added.
+    # Make sure we only keep enabled probes.
+    if editor.find("NP_PROBE") and "isEnabled" in editor.find("NP_PROBE").attrib.keys():
+        np_probes = [probe for probe in editor.findall("NP_PROBE") if probe.attrib["isEnabled"] == "1"]
```

Comment:
@vigji actually probably it's better to port the logic to https://github.com/SpikeInterface/probeinterface/pull/308/files#diff-9b4f228b41ad6dd338c7783897b5b949ef3825523b703d0376e755318f935549R1692-R1696

Depending on the Open Ephys version, the `NP_PROBE` field can end up in different locations of the settings.xml